project/plugins.sbt

```diff
@@ -34,7 +36,7 @@ addSbtPlugin(("com.lightbend.paradox" % "sbt-paradox-project-info" % "2.0.0").fo
 addSbtPlugin("com.github.sbt" % "sbt-unidoc" % "0.5.0")
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.2")
 addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.4.1")
-// Pekko gRPC -- sync with version in Dependencies.scala:29
-addSbtPlugin("org.apache.pekko" % "pekko-grpc-sbt-plugin" % "1.0.0")
+// Pekko gRPC -- sync with PekkoGrpcBinaryVersion in Dependencies.scala
```

Comment:
@pjfanning Also need to update it in `Dependencies.scala`

Reply:
I can't find anything to update in Dependencies.scala.

This is all I can find.
```
  // Sync with plugins.sbt
  val PekkoGrpcBinaryVersion = "current"
```

Do you want `current` changed to `1.0.1`?

Reply:
> I can't find anything to update in Dependencies.scala.
> 
> This is all I can find.
> ```
>   // Sync with plugins.sbt
>   val PekkoGrpcBinaryVersion = "current"
> ```
> 
> Do you want `current` changed to `1.0.1`? 

Hmm that sounds off for the same reasons as pekko http where other caused issues. I am getting the nagging feeling that we should leave this to 1.0.0 and setup a system where we build against snapshots as a nightly job

Reply:
Dependencies.PekkoGrpcBinaryVersion is only used in doc generation

Reply:
This was a big misunderstanding on my part, approving PR.